core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -273,12 +351,18 @@ object ConsumerGroupCommand {
       "information for the given consumer groups. For instance --group g1 --group g2 --topic t1" + nl +
       "Pass in just a topic to delete the given topic's partition offsets and ownership information " +
       "for every consumer group. For instance --topic t1" + nl +
-      "WARNING: Only does deletions on consumer groups that are not active."
+      "WARNING: Only does deletions on consumer groups that are not active." + nl +
```

Comment:
nit: "WARNING: Group deletion only works for old ZK-based consumer groups, and one has to use it carefully to only delete groups that are not active"